fastlane/Fastfile

```diff
@@ -94,6 +125,42 @@ platform :android do
     tag_release_with_latest_if_needed(release_version: release_version)
   end
 
+  desc "Make github release"
+  lane :github_amazon_release do |options|
+    release_version = options[:version]
+    fail ArgumentError, "missing version" unless release_version
+
+    begin
+      changelog = File.read("../CHANGELOG.latest.md")
+    rescue
+      fail "please add a CHANGELOG.latest.md file before calling this lane"
+    end
+
+    add_git_tag(
+      tag: "#{release_version}",
+      force: true,
+      grouping: "releases"
+    )
+
+    push_git_tags(
+      tag: "#{release_version}",
+      force: true
+    )
```

Comment:
I set the tag before creating the release, so it doesn't create a tag in main when creating the release and it does it in the tag